dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java

```diff
@@ -55,11 +55,11 @@ public boolean isDateInRange(OffsetDateTime timestamp) {
 	 * @return
 	 */
 	public boolean isValidKeyDate(Long keyDate) {
-		return (Instant.ofEpochMilli(keyDate).atOffset(ZoneOffset.UTC).getHour() == 0);
+		return keyDate != null && (Instant.ofEpochMilli(keyDate).atOffset(ZoneOffset.UTC).getHour() == 0);
```

Comment:
Not quite sure yet, but I would suggest changing keyDate to a `long` since in the RequestPath variable can never be null

Reply:
Good point. I switched `Long` to `long` for the PathVariables where the null value is not needed.